modules/project-docs/pages/compatibility.adoc

```diff
@@ -63,9 +63,10 @@ Other versions may work -- _but they are not tested and they are not supported_.
 
 The following JDK releases are supported at time of 3.4 SDK release (October 2022):
 
-* https://adoptium.net/[OpenJDK 11] (Hotspot recommended) or https://www.oracle.com/java/technologies/downloads/#jdk11[Oracle JDK 11]
-* https://adoptium.net/[OpenJDK 17 with HotSpot JVM] (recommended)
+* https://adoptium.net/[OpenJDK 21 with HotSpot JVM] (recommended)
+* https://adoptium.net/[OpenJDK 17 with HotSpot JVM]
 * https://www.oracle.com/java/technologies/downloads/#jdk17[Oracle JDK 17 (recommended)]
+* https://adoptium.net/[OpenJDK 11] (Hotspot recommended) or https://www.oracle.com/java/technologies/downloads/#jdk11[Oracle JDK 11]
 * https://adoptium.net/[OpenJDK 1.8 with HotSpot JVM]
 * https://www.oracle.com/java/technologies/downloads/#java8[Oracle JDK 1.8]
 
```

Comment:
Hey @ingenthr and @dnault I'm a touch unclear on our current support position w.r.t. Java - does this list look correct to you?